readthedocs/search/tests/data/generic/in/basic.html

```diff
@@ -6,5 +6,14 @@
   </head>
   <body>
     Content of the body.
+    <dl>
+      <dt id="love">Love</dt>
+      <dd>To code is to love</dd>
+      <dt>No section for me</dt>
+      <dd>This term does not have an ID so it's not gonna get its own section</dd>
+      <dt id="code">Code</dt>
+      <dt id="docs">Docs</dt>
+      <dd>Code and docs are like love, they are 4-letter words.</dd>
```

Comment:
Notice the two `<dt>` elements one after another: This is allowed, you can have several elements like that. Also in a Sphinx glossary.